### PR TITLE
jsbsim: add build from head

### PR DESCRIPTION
**Pull request by slnovak**
Allows installation from Sourceforge repository.

:beers:

### CODE REVIEW

**Comment by dpo** [comment]
Presumably, this needs a `revision` bump.

File: jsbsim.rb [hunk 1]
```diff
@@ -12,7 +12,23 @@ class Jsbsim < Formula
...
+    # remove tests since there's a test that also relies on "feenableexcept".
+    patch :DATA
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
   def install
+    system "./autogen.sh"
```
Comment (dpo) [inline jsbsim.rb]:
```
==> ./autogen.sh
**Warning**: I am going to run `configure' with no arguments.
If you wish to pass any to it, please specify them on the
`./autogen.sh' command line.
```
Looks like we can simply replace `configure` with `autogen.sh` below.

**Comment by slnovak** [comment]
Good eye, @dpo! I removed the call to `./configure` and am now passing the arguments directly to `./autogen.sh`. Verified both stable and head builds work.

Thanks!

**Comment by dpo** [comment]
Thanks. Could you please also add a `revision`? I'm assuming your changes change the bottles.

**Comment by slnovak** [comment]
Thanks @dpo! Added `revision 1` to the formula.

**Comment by dpo** [comment]
Looks like you restored the `autogen`/`configure` thing in the process.

**Comment by slnovak** [comment]
Gah -- sorry about that. Fixed, @dpo!

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**